### DISCUSSION TITLE
Menu Bar Item?

### DISCUSSION THREAD

**Discussion by michelef** [gingerbeardman/stapler#39] [Ideas]
Hi,

I think tat having a menubar item would be great.
It could just have a list of all the windows' names , by date or alphabetical order (selectable), and just choosing that item in the menubar would directly open the docs/apps

**Comment by gingerbeardman** [comment]
Interesting idea. 

What benefits do you think it would offer over the app's own Window or File > Recent menu items?

**Comment by michelef** [comment]
Matt,

You would not have to switch to the app and have it available everywhere.

Slight improvement but it would be quicker. I really like all the apps that have Menu bar items.

Thanks for considering it.

Best,
Michele



--
Michele Fuortes, M.D., Ph.D.
ITS Departmental Liaison for Research
Assistant Professor
Department of Cell and Developmental Biology
Weill Cornell Medical College
E-mail: ***@***.***
Phone (212) 746-6188
Cell (646) 628-7958
Room OH-1009

**Reply by gingerbeardman** [reply]
No worries, I will look at how much is involved in creating a menu bar item and if it can be hosted inside the same app. If not, I don't have the time to create a second app for that. I'll post back here if seems possible.
